src/test/java/gov/healthit/chpl/aqa/pageObjects/ProductHistoryEyePage.java

```diff
@@ -0,0 +1,27 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ProductHistoryEyePage definition.
+ */
+public final class ProductHistoryEyePage {
```

Comment:
The URL for where this page is now uses /listing, and not product. Recommend changing to use Listing instead of Product here, that way in the WebElement productHistoryProduct you can make it look reasonable by using listingHistoryProduct